SpeechService/SpeechService.cs

```diff
@@ -352,7 +363,7 @@ private void speak(MemoryStream stream, string voice, string speech)
                     }
                     Logging.Debug("Configuration is " + Configuration == null ? "<null>" : JsonConvert.SerializeObject(Configuration));
                     synth.Rate = Configuration.Rate;
-                    synth.Volume = Configuration.Volume;
+                    synth.Volume = (int)Math.Round(Configuration.Volume * (transmitVolume / 100));
```

Comment:
Hmm... ok.
What if I want normal volume at 50% and transmit volume in some circumstances to be at 60%
Why make the `transmitVolume` dependent on `Configuration.Volume` rather than separate?
e.g.
```cs
public void Say(Ship ship, string message, int priority = 3, string voice = null, bool radio = false, string eventType = null, bool invokedFromVA = false, int? volumeOverride = null)
// ...
synth.Volume = (int)(volumeOverride ?? Configuration.Volume); 
```